src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -21,6 +21,32 @@
 
 namespace Tizen.NUI.Components
 {
+    /// <summary>
+    /// Slider value changed event data.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class SliderValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Curren value
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public float CurrentValue;
+    }
+
+    /// <summary>
+    /// Slider sliding finished event data.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class SliderSlidingFinishedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Curren value
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        public float CurrentValue;
```

Comment:
Please DO NOT use public field variables. Use public property instead. Also does it have to be a public member? 
```suggestion
        public float CurrentValue { get; set; }
```